Predictors/3d_kPCA.py

```diff
@@ -0,0 +1,48 @@
+import matplotlib.pyplot as plt
+# This import is used even though PyCharm thinks it is not
+# In case deleted it is this import "from mpl_toolkits.mplot3d import Axes3D"
```

Comment:
The standard way of doing this is by telling pycharm's linter to ignore it:
```
# noinspection PyUnresolvedReferences
```
it's a bad idea to put it in a comment in case it's deleted, as you could easily update one without updating the other.

Reply:
(this should also fix the CI linter too, as it's pretty standard)

Reply:
It didn't fix the CI but we can just ignore that